rules/camunda-cloud/version-compatibility-validator.js

```diff
@@ -0,0 +1,144 @@
+const {
+  findExtensionElement
+} = require('../utils/element');
+
+const { reportErrors } = require('../utils/reporter');
+
+const { skipInNonExecutableProcess } = require('../utils/rule');
```

Comment:
I'm not sure if this fully utilizes the existing infrastructure we have in place. I asked @philippfromme to review, too.

Reply:
👀